src/seismometer/plot/mpl/_ux.py

```diff
@@ -200,11 +199,11 @@ def _register_colormaps() -> None:
     text_cmap = LinearSegmentedColormap.from_list("Text", text_colors, N=len(text_colors))
     alert_cmap = LinearSegmentedColormap.from_list("Alert", alert_colors, N=len(alert_colors))
     neutral_cmap = LinearSegmentedColormap.from_list("Neutral", neutral_colors, N=len(neutral_colors))
-    plt.register_cmap(cmap=area_cmap)
-    plt.register_cmap(cmap=line_cmap)
-    plt.register_cmap(cmap=text_cmap)
-    plt.register_cmap(cmap=alert_cmap)
-    plt.register_cmap(cmap=neutral_cmap)
+    mpl.colormaps.register(cmap=area_cmap)
```

Comment:
Adding force=True here will make it safe to run multiple times (at the risk of overwriting someone else's names)

Reply:
FINC